### ISSUE TITLE
upgrade to webpack2

### ISSUE THREAD

**Issue by scottdraves** [twosigma/beakerx#5408]
```
warning "sass-loader@6.0.5" has incorrect peer dependency "webpack@^2.0.0".
warning "watch-ignore-webpack-plugin@1.0.0" has incorrect peer dependency "webpack@1.5.x".
```
are these from upstream?

**Comment by MariuszJurowicz** [comment]
@scottdraves 
The `sass-loader` version `6` has a peer dependency set for `webpack@^2.0.0` to remove this warning we can use the version `4.x` of `sass-loader` that allows the webpack `1`.

If we use webpack `1.15.x` and `watch-ignore-webpack-plugin` the second warning will continue to show as there is peer dependency set to fixed version webpack `1.5.x`.
We can fix all issues by migrating to webpack 2. Then we can drop the `watch-ignore-webpack-plugin` as the file watcher can be configured in webpack 2 to ignore specified directories.

**Comment by scottdraves** [comment]
thanks, so let's migrate.
